integration/env/azure.go

```diff
@@ -80,7 +80,8 @@ func init() {
 	var ok bool
 	sshPublicKey, ok = os.LookupEnv(EnvVarBastionPublicSSHKey)
 	if !ok {
-		fmt.Printf("No value found in '%s': no keys will be placed on the bastion server\n", EnvVarBastionPublicSSHKey)
+		fmt.Printf("No value found in '%s': default public key will be placed on the bastion server\n", EnvVarBastionPublicSSHKey)
+		sshPublicKey = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDBSSJCLkZWhOvs6blotU+fWbrTmC7fOwOm0+w01Ww/YN3j3j1vCrvji1A4Yonr89ePQEQKfZsYcYFodQI/D3Uzu9rOFy0dCMQfvL/J6N8LkNtmooh3J2p061829MurAdD+TVsNGrD2FZGm5Ab4NiyDXIGAYCaHL6BHP16ipBglYjLQt6jVyzdTbYspkRi1QrsNFN3gIv9V47qQSvoNEsC97gvumKzCSQ/EwJzFoIlqVkZZHZTXvGwnZrAVXB69t9Y8OJ5zA6cYFAKR0O7lEiMpebdLNGkZgMA6t2PADxfT78PHkYXLR/4tchVuOSopssJqgSs7JgIktEE14xKyNyoLKIyBBo3xwywnDySsL8R2zG4Ytw1luo79pnSpIzTvfwrNhd7Cg//OYzyDCty+XUEUQx2JfOBx5Qb1OFw71WA+zYqjbworOsy2ZZ9UAy8ryjiaeT8L2ZRGuhdicD6kkL3Lxg5UeNIxS2FLNwgepZ4D8Vo6Yxe+VOZl524ffoOJSHQ0Gz8uE76hXMNEcn4t8HVkbR4sCMgLn2YbwJ2dJcROj4w80O4qgtN1vsL16r4gt9o6euml8LbmnJz6MtGdMczSO7kHRxirtEHMTtYbT1wNgUAzimbScRggBpUz5gbz+NRE1Xgnf4A5yNMRy+JOWtLVUozJlcGSiQkVcexzdb27yQ=="
```

Comment:
What/which/whose key is this?

Reply:
In order to be able to run the tests from the command line without having to pass the SSH public key as an argument, I need to set a default one. I used mine. The best approach would be to fetch all of them from the installations repository but... not enough time.